titus-api/src/main/java/com/netflix/titus/api/clustermembership/connector/mixin/ClusterMembershipRevisionMixIn.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.api.clustermembership.connector.mixin;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.netflix.titus.api.clustermembership.model.ClusterMember;
+
+public abstract class ClusterMembershipRevisionMixIn {
```

Comment:
add `@JsonIgnoreProperties(ignoreUnknown = true)` preemptively to allow backwards compatible changes later

Reply:
I will remove this mixin (no longer needed).